src/input/accessors.cc

```diff
@@ -48,7 +48,7 @@ const char * Exception::what() const throw () {
         converter << boost::diagnostic_information_what( *this );
         print_stacktrace(converter);
```

Comment:
Introduce a method:
	virtual ostream &ExceptionBase::form_message(ostream &);

Use it in the what message.
Reimplement it in the Input::Exception and possibly in Exc_Assert.